libs/sql-schema-describer/Cargo.toml

```diff
@@ -7,6 +7,7 @@ version = "0.1.0"
 [dependencies]
 native-types = { path = "../native-types" }
 prisma-value = { path = "../prisma-value" }
+datamodel = { path = "../datamodel/core" }
```

Comment:
(haven't read the rest of the PR yet) — why datamodel here? I'm not comfortable with this.

Reply:
no need. taken out.